.github/workflows/experimental.yml

```diff
@@ -54,13 +54,55 @@ jobs:
       DOCKER_IMAGE_TAG: ${{ matrix.version }}
       DOCKER_IMAGE_WEBLOG_TAG: ${{ github.sha }}
       BUILDX_PLATFORMS: linux/amd64
+      MODE: manual
```

Comment:
I am not sure if we could remove "MODE" env parameter. We know that we are executing "run-manual-lib-injection.sh"

Reply:
This variable is needed to define unique artefact names in `lib-injection/runner/action.yml`, happy to change it if there is a better alternative!